src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -9603,6 +9695,69 @@ App::DocumentObject *SketchObject::getSubObject(
     return const_cast<SketchObject*>(this);
 }
 
+std::vector<Data::IndexedName>
+SketchObject::getHigherElements(const char *element, bool silent) const
+{
+    std::vector<Data::IndexedName> res;
+        if (boost::istarts_with(element, "vertex")) {
+            int n = 0;
+            int index = atoi(element+6);
+            for (auto cstr : Constraints.getValues()) {
+                ++n;
+                if (cstr->Type != Sketcher::Coincident)
+                    continue;
+                for (int i=0; i<2; ++i) {
+                    int geoid = i ? cstr->Second : cstr->First;
+                    const Sketcher::PointPos &pos = i ? cstr->SecondPos : cstr->FirstPos;
+                    if(geoid >= 0 && index == getSolvedSketch().getPointId(geoid, pos) + 1)
+                        res.push_back(Data::IndexedName::fromConst("Constraint", n));
+                };
+            }
+        }
+        return res;
+
+    auto getNames = [&](const char *element) {

```

Comment:
The CI linter doesn't complain about it, but mine does --Clang-Tidy: Lambdas that implicitly capture 'this' should not specify a capture default"
```suggestion
    auto getNames = [this, &silent, &res](const char *element) {
```